### ISSUE TITLE
Apply MinNet for custom analyses

### ISSUE THREAD

**Issue by JianiC** [ChaozhongLiu/MinNet#3]
Hello,
After successfully testing the module you provided, I am excited to apply the same approach to a custom dataset containing both paired and unpaired batches. I have successfully executed the training code. However, I do have a couple of inquiries that I hope you could assist me with:

1. In the context of the '**MinNet/train/10X/2_training_main.ipynb'** notebook, you mentioned substituting the '**SiaAtt.ckpt'** file in my submission with the '**ckpts/model_margin10.0_lamb0.5_epoch35.ckpt**' file. However, I noticed in the '**MinNet/fig2/10XBrain/SiaNN.ipynb'** notebook, the '**utils/SiaNN.ckpt**' was loaded. Could you kindly guide me through the steps required to derive 'SiaKK.ckpt' from the previously mentioned 'SiaAtt.ckpt'? 

2. I couldn't locate specific details regarding the preprocessing of the training data in your manuscript. In the '**MinNet/blob/master/train/10X/1_training_data_processing.ipynb**' notebook, Could you clarify if the two input h5 files are raw counts or based on normalized data?

**Comment by ChaozhongLiu** [comment]
Hi there,

Thanks for your interest in our model! And I would be glad if MinNet could help with your research.

1. When you trained your model in ``2_training_main.ipynb``, just check the *.ckpt files in your ``ckpt`` folder to find the exact name you need. Then all you need to do is move the final checkpoint file to ``utils/`` folder when you apply it in your unpaired dataset. In ``MinNet/fig2/10XBrain/SiaNN.ipynb``: ``ckpt_loader = torch.load('utils/SiaNN.ckpt')``, change the file path to the name of your checkpoint file.

2. Sorry for the confusion. To clarify, it is the library size-normalized and log-transformed data.

Let me know if you have any other questions!
